pkg/eks/api.go

```diff
@@ -137,10 +137,12 @@ func (c *ClusterProvider) runCreateTask(tasks map[string]func(chan error) error,
 			logger.Debug("task %q started", tn)
 			errs := make(chan error)
 			if err := task(errs); err != nil {
+				logger.Debug("task %q failed to start due to error %#v", tn, err)
```

Comment:
Do we need these log statements? Won't the errors be logged at the higher level anyway.

Reply:
Not needed. I'll take them out. I just really wanted to know when debugging, the difference between a returned error, vs a channel-read error. Sorry about this.